python/edit_fails_by_row.py

```diff
@@ -0,0 +1,89 @@
+import math
+import json
+import os
+import pandas as pd
+import random
+import string
+import time
+import yaml
+import utils
+
+from lar_generator import lar_gen #Imports lar_gen class. 
+
+from large_test_files import LargeTestFiles
+
+from large_test_files import unique_uli
+
+class EditFailsByRow(object):
+
+	def __init__(self, passes_all_filepath, passes_all_filename, 
+		fails_all_filepath, fails_all_filename):
+		"""The class instantiates with a filepath and name to a file that 
+		passes every edit, and a filepath and name to a file that fails 
+		a an edit or multiple edits for every row."""
+		
+		#Loading TS and LAR data for the clean file. 
+		self.ts_data, self.passes_all_lar = utils.read_data_file(path=passes_all_filepath, 
+                                                 data_file=passes_all_filename)
+
+		#Loading TS and LAR data for the file that fails a set of edits for every row. 
+		other_ts_data, self.fails_all_lar = utils.read_data_file(path=fails_all_filepath,
+												 data_file=fails_all_filename)
+
```

Comment:
Lets add a comment that describes the why of setting up all these data elements.